src/targets/shell/curl.js

```diff
@@ -39,6 +40,22 @@ module.exports = function (source, options) {
     code.push(opts.short ? '-0' : '--http1.0')
   }
 
+  // if multipart form data, we want to remove the boundary
```

Comment:
Per #226, we discussed just scoping this to fixing cURL due to how cURL handles `--form` vs raw data.

Did you attempt any kind of audit of other targets to see if they are impacted by similar issues?

Reply:
I have not investigated any other targets yet, this fix only pertains to curl multipart